src/inc/test/CommonState.hpp

```diff
@@ -256,49 +256,20 @@ class CommonState
     std::unique_ptr<TextBuffer> m_backupTextBufferInfo;
     std::unique_ptr<INPUT_READ_HANDLE_DATA> m_readHandle;
 
-    struct TestString
-    {
-        std::wstring_view string;
-        bool wide = false;
-    };
-
-    static void applyTestString(ROW* pRow, const auto& testStrings)
-    {
-        uint16_t x = 0;
-        for (const auto& t : testStrings)
-        {
-            if (t.wide)
-            {
-                pRow->ReplaceCharacters(x, 2, t.string);
-                x += 2;
-            }
-            else
-            {
-                for (const auto& ch : t.string)
-                {
-                    pRow->ReplaceCharacters(x, 1, { &ch, 1 });
-                    x += 1;
-                }
-            }
-        }
-    }
-
     void FillRow(ROW* pRow, bool wrapForced)
     {
         // fill a row
         // 9 characters, 6 spaces. 15 total
         // か = \x304b
         // き = \x304d
 
-        static constexpr std::array testStrings{
-            TestString{ L"AB" },
-            TestString{ L"\x304b", true },
-            TestString{ L"C" },
-            TestString{ L"\x304d", true },
-            TestString{ L"DE      " },
-        };
-
-        applyTestString(pRow, testStrings);
+        uint16_t column = 0;
+        for (const auto& ch : std::wstring_view{ L"AB\u304bC\u304dDE      " })
+        {
+            const uint16_t width = ch >= 0x80 ? 2 : 1;
+            pRow->ReplaceCharacters(column, width, { &ch, 1 });
+            column += width;
+        }

```

Comment:
I forgot why I made this change... But it's much shorter now, so there's that at least.